Lesson3/src/Hometask.java

```diff
@@ -0,0 +1,118 @@
+import com.sun.xml.internal.bind.v2.TODO;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Hometask {
+    public static void main(String[] args) {
+        System.out.println(operation(1));
+        System.out.println(operation(0));
+        foobar(6);
+        foobar(10);
+        foobar(15);
+        printMatrix();
+        printPrimeNumbers();
+    }
+
+    /**
+     * Method should perform some operation with int "number" based on some conditions:
+     * - if number is positive - increase it by 1
+     * - if number is negative - decrease it by 2
+     * - if number is equal to zero - replace it with 10
+     * return number value after operation
+     */
+    public static int operation(int number) {
+        if (number > 0) {
+            number++;
+        }
+        if (number < 0) {
+            number = number - 2;
+        }
+        if (number == 0) {
+            number = 10;
+        }
+        return number;
+    }
+
+    /**
+     * Method should print different strings in console based on some conditions:
+     * - if remainder of the division number by 3 is zero - print "foo" (example of number - 6)
+     * - if remainder of the division number by 5 is zero - print "bar" (example of number - 10)
+     * - if remainder of the division number by both 3 and 5 is zero 0  and - print "foobar" (example of number - 15)
+     */
+    public static void foobar(int number) {
```

Comment:
if number will be = 15 this method will print
foo
bar
foobar

but it should print only foobar